### PR TITLE
Reduce exec frequency of legacy jobs

### PR DESCRIPTION
**Pull request by jmguzik**
Used modified script: https://github.com/bear-redhat/tools/issues/1 (sorry not PR)
/cc @jupierce @bear-redhat 
Signed-off-by: Jakub Guzik <jguzik@redhat.com>

### CODE REVIEW

**Comment by jmguzik** [comment]
ref: https://github.com/openshift/release/pull/31193

File: ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 periodics:
 - agent: kubernetes
   cluster: build03
+  cron: 41 3 */14 * *
```
Comment (jupierce) [inline ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml:4]: We can fix these up in any way you see fit. In this PR or after the fact, but `promote-...machine-os-content..` jobs need to continue to run often. They are more of an infra job than a test.

Comment (bear-redhat) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml:4]: ill address this in the script to skip `promote-*`

Comment (jmguzik) [reply ci-operator/jobs/openshift/release/openshift-release-release-4.8-periodics.yaml:4]: Ack, I think we are close finishing this task, I'll leave promotion jobs to you @bear-redhat

**Pull request closed without merging**